apps/meteor/ee/client/voip/modals/DeviceSettingsModal.tsx

```diff
@@ -60,6 +60,11 @@ const DeviceSettingsModal = (): ReactElement => {
 						</Box>
 					</Box>
 				)}
+				{!isSecureContext && (
```

Comment:
c'mon provide some error/info from the provider and dont spread the `isSecureContext` thats why you create this provider thats right?